test/integration/smoke-test.ts

```diff
@@ -23,3 +23,13 @@ test('require("@octokit/webhooks").createEventHandler', () => {
   }).not.toThrow();
   expect(emitWarningSpy).not.toHaveBeenCalled();
 });
+
+test('require("@octokit/webhooks").emitterEventNames', () => {
+  const emitWarningSpy = jest.spyOn(process, "emitWarning");
+
+  const allEvents = emitterEventNames;
+  expect(typeof allEvents).toBe("object");
+  expect(typeof allEvents[0]).toBe("string");
+
+  expect(emitWarningSpy).not.toHaveBeenCalled();
```

Comment:
no need to spy on process.emitWarning for this test

```suggestion
  const allEvents = emitterEventNames;
  expect(typeof allEvents).toBe("object");
  expect(typeof allEvents[0]).toBe("string");
```